internal/rpc/starknet/types.go

```diff
@@ -552,3 +552,7 @@ func NewStateUpdate(s *types.StateUpdate) *StateUpdate {
 		StateDiff: stateDiff,
 	}
 }
+
+type Status struct {
+	Status bool `json:"status"`
```

Comment:
If we are going to return a boolean value then let's change the key to make it more informative i.e. `"available"` so that the response reads:

```json
{
     "jsonrpc": "2.0",
     "result": {
         "available": true,
 	},
     "id": 1
 }
```